Build/Chakra.Build.Paths.props

```diff
@@ -11,9 +11,9 @@
   </PropertyGroup>
   <PropertyGroup Label="ICU">
     <!-- IcuLibDir can be set in environment or with "/p:IcuLibDir=..." -->
-    <IcuLibDir Condition="'$(IcuLibDir)'=='' AND '$(Configuration)'=='Debug'">$(ChakraCoreRootDirectory)\deps\static\icu-small\debug</IcuLibDir>
-    <IcuLibDir Condition="'$(IcuLibDir)'=='' AND ('$(Configuration)'=='Test' or '$(Configuration)'=='Release')">$(ChakraCoreRootDirectory)\deps\static\icu-small\release</IcuLibDir>
+    <!-- <IcuLibDir Condition="'$(IcuLibDir)'=='' AND '$(Configuration)'=='Debug'">$(ChakraCoreRootDirectory)\deps\static\icu-small\debug</IcuLibDir> -->
```

Comment:
What is the purpose of keeping these lines around commented out? Are they intended as documentation of how to specify the `IcuLibDir` parameter to msbuild?

Reply:
The real reason is because I commented them out to ensure nothing was being set behind my back and then made the pull request without remembering to either comment their removal better or add them back. My comment above asking doug what to do here is why it hasn't been fixed yet.

Reply:
Yeah, let's keep them for now.